Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -278,7 +278,7 @@ open class RouteController: NSObject {
         updateRoadName(status: status)
         
         if willReroute {
-            reroute(from: location, along: routeProgress)
+            reroute(from: CLLocation(status.location), along: routeProgress)
```

Comment:
Is it enough to use snapped location only for this case? I can see that there are also a couple of places in this method, where it'd be preferable to use snapped location: e.g. we check whether reroute will be performed using raw location, but to make a request we use snapped one.